src/Sparse/Row.jl

```diff
@@ -447,11 +450,32 @@ end
 #  Inplace scaling
 #
 ################################################################################
+@doc raw"""
+    scale_row!(a::SRow, b::NCRingElem) -> SRow
 
+Returns the (left) product of $b \times a$.
+For rows, the standard multiplication is from the left. 
+"""
 function scale_row!(a::SRow{T}, b::T) where T
   @assert !iszero(b)
   if isone(b)
-    return
+    return a
+  end
+  for i=1:length(a.pos)
+    a.values[i] = b*a.values[i]
```

Comment:
As we just discussed orally: The problem is that in the meantime PR #1476 was merged which made a fix to `scale_row!` over rings with zero divisors, and this fix now ended up being only in `scale_row_right!` but not in this method.